schemas/1.3.0/adaptive-card.json

```diff
@@ -982,19 +938,19 @@
         },
         "max": {
           "type": "string",
-          "description": "Hint of maximum value expressed in ISO-8601 format (may be ignored by some clients)."
+          "description": "Hint of maximum value expressed in YYYY-MM-DD(may be ignored by some clients)."
```

Comment:
This should indeed be ISO8601. It has to support a time offset, otherwise it will not work properly across time zones.

Reply:
Should we remove "may be ignored by some clients"?

Reply:
This wasn't something i intentionally changed. It changed as a result of re-generating the docs.

That said, I'm pretty sure at least UWP is only supporting is only supporting YYYY-MM-DD for all fields in Input.Date. We had a whole discussion about this at some point, let me see if I can dig it up. 

---
In reply to: [447778889](https://github.com/microsoft/AdaptiveCards/pull/4255#discussion_r447778889) [](ancestors = 447778889)

Reply:
So, this YYYY-MM-DD format is what's currently showing up on the website schema explorer. (Presumably because the source files were already showing that, not sure why this 1.3 version isn't)

As for this history of this decision, I couldn't quite find the smoking gun, but here's what i've tracked down. In February 2019 we had a long mail thread about what to do with this scenario, which ended in us deciding to have a meeting and talk about it in person. I didn't find meeting notes, but following that meeting I implemented the YYYY-MM-DD (and HH:MM for time) behavior in UWP (#2450) and we logged several bugs to standardize on YYYY-MM-DD and HH:MM for Input.Date and Input.Time (#2495, #2496, #2497). 

All of which is to say i believe the current spec is for YYYY-MM-DD for Input.Date (inline dates in TextBlocks are different) and this apparent change showing up in my PR is an artifact of the 1.3 version being out of date for some reason (The source file was changed in January as part of #3706). If we want to talk about the merits of that decision, that's probably outside the scope of this PR.

---
In reply to: [448480840](https://github.com/microsoft/AdaptiveCards/pull/4255#discussion_r448480840) [](ancestors = 448480840,447778889)

Reply:
Well, that simply doesn't work... Just for context, we just uncovered a bug in Outlook a couple weeks ago, check out the details here: https://github.com/microsoft/AdaptiveCards/issues/4186

Any time we are in the business of manipulating dates, we must support a time offset.

Looks like we should have yet another conversation about this.